symfony/client/src/orangehrmAdminPlugin/pages/employmentStatus/EmploymentStatus.vue

```diff
@@ -0,0 +1,179 @@
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-paper-container">
+      <div class="orangehrm-header-container">
+        <oxd-text tag="h6">Employment Status List</oxd-text>
+        <div>
+          <oxd-button label="Add" displayType="secondary" @click="onClickAdd" />
+        </div>
+      </div>
+      <oxd-divider class="orangehrm-horizontal-margin" />
+      <div>
+        <div class="orangehrm-horizontal-padding orangehrm-vertical-padding">
+          <div v-if="checkedItems.length > 0">
+            <oxd-text tag="span">
+              {{ checkedItems.length }} Employment Status Selected
+            </oxd-text>
+            <oxd-button
+                label="Delete Selected"
+                displayType="label-danger"
+                @click="onClickDeleteSelected"
+                class="orangehrm-horizontal-margin"
+            />
+          </div>
+          <oxd-text tag="span" v-else> {{ total }} Employment Status Found</oxd-text>
+        </div>
+      </div>
+      <div class="orangehrm-container">
+        <oxd-card-table
+            ref="dTable"
+            :headers="headers"
+            :items="items?.data"
+            :selectable="true"
+            v-model:selected="checkedItems"
+            rowDecorator="oxd-table-decorator-card"
+        />
+      </div>
+      <div class="orangehrm-bottom-container">
+        <oxd-pagination
+            v-if="showPaginator"
+            :length="pages"
+            v-model:current="currentPage"
+        />
+      </div>
+    </div>
+
+    <delete-confirmation ref="deleteDialog"></delete-confirmation>
+  </div>
+</template>
+
+<script>
+import usePaginate from '@/core/util/composable/usePaginate';
+import DeleteConfirmationDialog from '@orangehrm/components/dialogs/DeleteConfirmationDialog';
+import {navigate} from "@/core/util/helper/navigation";
+
+export default {
+  data() {
+    return {
+      headers: [
+        {name: 'name', title: 'Employment Status', style: {'flex-basis': '80%'}},
+        {
+          name: 'actions',
+          title: 'Actions',
+          style: {'flex-shrink': 1},
+          cellType: 'oxd-table-cell-actions',
+          cellConfig: {
+            delete: {
+              onClick: this.onClickDelete,
+              component: 'oxd-icon-button',
+              props: {
+                name: 'trash',
+              },
+            },
+            edit: {
+              onClick: this.onClickEdit,
+              props: {
+                name: 'pencil-fill',
+              },
+            },
+          },
+        },
+      ],
+      editItem: null,
+      checkedItems: [],
+    };
+  },
+
+  components: {
+    'delete-confirmation': DeleteConfirmationDialog,
+  },
+
+  setup() {
+    const {
+      showPaginator,
+      currentPage,
+      total,
+      pages,
+      pageSize,
+      response,
+      isLoading,
+      execQuery,
+    } = usePaginate('api/v1/admin/employment-statuses');
```

Comment:
`usePaginate` has since been updated to accept an APIService instance. Please follow [this](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmAdminPlugin/pages/jobCategory/JobCategory.vue#L133)